queries.sql

```diff
@@ -15,3 +15,52 @@ SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- TRANSACTIONS
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
Hi,

Great job creating queries to answer the following questions in transactions. I have reviewed your code and found one issue that needs to be fixed before your code can meet the project requirements. Here is my comment and suggestion:

- Please add another `SELECT * FROM animals;` statement before your `ROLLBACK` statement. This will show you the number of records in the animals table after you deleted them, and verify that the change was made. Then, you can compare it with the number of records after you rolled back the transaction, and verify that the change was undone. This will match the project requirement of verifying that the change was made and rolled back.

Please make this change and resubmit your code for review. If you have any questions or need any help, please let me know.

Happy coding!